include/sdf/JointAxis.hh

```diff
@@ -31,11 +31,34 @@ namespace sdf
 {
   // Inline bracket to help doxygen filtering.
   inline namespace SDF_VERSION_NAMESPACE {
-  //
 
   // Forward declare private data class.
   struct PoseRelativeToGraph;
   template <typename T> class ScopedGraph;
+  /// \brief Helper class to hold contents of the mimic joint tag.
+  /// The parent and child joint positions are related with:
+  /// multiplier * (parentJoint - reference) + offset = childJoint.
+  class SDFORMAT_VISIBLE mimicJoint
```

Comment:
If this class is visible, should we use the PIMPL pattern?

Reply:
Done https://github.com/gazebosim/sdformat/pull/1166/commits/8ad2358ac490e19aea7d07f188e4108de984c19e